kokoro/build_binaries_rust.sh

```diff
@@ -30,8 +30,8 @@ touch "${KOKORO_ARTIFACTS_DIR}/binaries/git_commit_${KOKORO_GIT_COMMIT_oak:?}"
 # Copy the generated binaries to Placer. The timestamps are used to convey
 # the creation time.
 readonly generated_binaries=(
-    ./oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/oak_restricted_kernel_simple_io_wrapper_bin
-    ./oak_restricted_kernel_wrapper/target/x86_64-unknown-none/release/oak_restricted_kernel_simple_io_init_rd_wrapper_bin
+    ./oak_restricted_kernel_wrapper/bin/oak_restricted_kernel_simple_io/oak_restricted_kernel_simple_io_bzimage
```

Comment:
Is this correct ? I thought it ends in "wrapper_bin" not in "bzimage" ?

Reply:
good catch! thank you.